code/controllers/configuration.dm

```diff
@@ -91,6 +91,7 @@ var/list/gamemode_cache = list()
 	var/allow_extra_antags = 0
 	var/guests_allowed = TRUE
 	var/debugparanoid = 0
+	var/evac_min_time = 0
```

Comment:
This doesn't belong in config, game balance shouldn't be there at all. I'm gradually phasing out existing balance aspects.